src/config.rs

```diff
@@ -1,118 +1,174 @@
-use crate::keystore;
-
-use arrayref::array_ref;
-use ed25519_dalek::*;
+// 
+// Manages creation, serialization (storage) and deserialization (loading) of HoloPort Device Seed
+// and Admin keys.
+// 
+// The HoloPort's /var/lib/holo-config.json Config is loaded, and the Config::V1.seed is used to
```

Comment:
More likely, `/run/keys/holo-config.json`, where `/run/keys` is tmpfs.

Reply:
Ah... We'll probably be loading this config from temporarily mounted storage; the USB drive, and then unmounting it (and then black-listing the USB device it rode in on, and preventing loading of any other modules to prevent anyone from re-accessing it).  So, we'll probably not have it available at any "standard" location.